content/en/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -197,23 +197,72 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
 
 1. The container runtime configures each Windows container with the specified GMSA credential spec so that the container can assume the identity of the GMSA in Active Directory and access services in the domain using that identity.
 
+## Containerd 
+
+On Windows Server 2019, in order to use GMSA with Containerd, you must be running OS Build 17763.1817 (or later) which can be installed using the patch [KB5000822](https://support.microsoft.com/en-us/topic/march-9-2021-kb5000822-os-build-17763-1817-2eb6197f-e3b1-4f42-ab51-84345e063564). 
+
+There is also a known issue with containerd when trying to connect to SMB shares from Pods. Once you have configured GMSA, the pod will be unable to connect to the share using the Hostname or FQDN but connecting via IP address works as expected. 
+
+```PowerShell
+ping adserver.ad.local
+```
+Resolves the Hostname to IP Address Correctly and will produce an output similar to this:
+```
+```PowerShell
```

Comment:
@perithompson please omit the “PowerShell” from this line - it's output, not PowerShell.